src/stores/transactions.ts

```diff
@@ -152,9 +212,11 @@ export class Transaction {
   amount: RawAmount;
   amountFee: RawAmount;
   fee: RawAmount;
+  /** TODO may be a problem when keeping the same transaction for 2 users */
```

Comment:
You mean when both the sender account and recipient account are in the same wallet? That worked as expected while I was testing things.

Reply:
Note for the future, related to #191.